libcxx/test/std/numerics/complex.number/cmplx.over/pow.pass.cpp

```diff
@@ -102,5 +102,8 @@ int main(int, char**)
     test<long double, float>();
     test<long double, double>();
 
-  return 0;
+    // Make sure LWG4191: P1467 is respected.
```

Comment:
I don't think we need to change this file either. `test<int, float>()`, `test<unsigned, float>()`, and `test<long long, float>()` should have been sufficient.